README.md

```diff
@@ -1 +1,287 @@
+
+```bash
+$ ballerina build guide/hello_world_service.bal
+Compiling source
+    hello_world_service.bal
+Generating executable
+    hello_world_service.balx
+        @kubernetes:Service                      - complete 1/1
+        @kubernetes:Deployment                   - complete 1/1
+        @kubernetes:Docker                       - complete 2/3
+        @kubernetes:Docker                       - complete 3/3
+        @kubernetes:Helm                         - complete 1/1
+
+        Run the following command to deploy the Kubernetes artifacts:
+        kubectl apply -f /home/manurip/Documents/Work/Repositories/ballerina-gke-deployment/kubernetes/
+
+        Run the following command to install the application using Helm:
+        helm install --name hello-world-service-deployment /home/manurip/Documents/Work/Repositories/ballerina-gke-deployment/kubernetes/hello-world-service-deployment
+```
+
+After the build is complete, the Docker image is created and pushed to Docker Hub. The Kubernetes deployment artifacts are generated as well.
+
+## Deployment
+
+- Configuring GKE environment
+
+Before deploying the service on GKE, you will need to setup the GKE environment to create the Kubernetes cluster and deploy an application.
+
+Let's start by installing the Google Cloud SDK in our local machine. Please refer to [Google Cloud SDK Installation](https://cloud.google.com/sdk/install) in finding the steps for the installation.
+
+Next step is gcloud configuration and creating a Google Cloud Platform project.
+You can begin with `gcloud init` command.
+
+```bash
+$ gcloud init
+Welcome! This command will take you through the configuration of gcloud.
+
+Your current configuration has been set to: [default]
+
+You can skip diagnostics next time by using the following flag:
+  gcloud init --skip-diagnostics
+
+Network diagnostic detects and fixes local network connection issues.
+Checking network connection...done.
+Reachability Check passed.
+Network diagnostic passed (1/1 checks passed).
+
+You must log in to continue. Would you like to log in (Y/n)?
+
+Your browser has been opened to visit:
+
+    https://accounts.google.com/o/oauth2/auth?redirect_uri=http%3A%2F%2Flocalhost%3A8085%2F&prompt=select_account&response_type=code&client_id=32555940559.apps.googleusercontent.com&scope=https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fuserinfo.email+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fappengine.admin+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcompute+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Faccounts.reauth&access_type=offline
+
+```
+
+Once you click on the provided link, you will be prompted to login to your Google account and authorize.
+
+Once authentication flow completes, you will be prompted to create a new project or choose an existing project as the
+current project. Here, we are creating a project named "BallerinaDemo"
```

Comment:
Remove line break and add a period at the end of the sentence. Apply elsewhere if applicable.